honeybadgermpc/ntl/ccobject.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * Credits to sagemath
```

Comment:
Add slightly more detail - just in case we want to track this down and make a more formal reference?

Reply:
Added a link to the original file. Should I add anything else?